docs/content/latest/api/ysql/datatypes/type_datetime/functions/_index.md

```diff
@@ -0,0 +1,147 @@
+---
+title: General-purpose date and time functions [YSQL]
+headerTitle: General-purpose date and time functions
+linkTitle: General-purpose functions
+description: Describes the general-purpose date and time functions. [YSQL]
+image: /images/section_icons/api/subsection.png
+menu:
+  latest:
+    identifier: date-time-functions
+    parent: api-ysql-datatypes-datetime
+    weight: 90
+isTocNested: true
+showAsideToc: true
+---
+
+This page lists all of the general-purpose _date-time_ functions. They are classified into groups according to the purpose.
+
+- [Creating date-time values](#functions-for-creating-date-time-values-creating-date-time-values)
+- [Manipulating date-time values](#functions-for-manipulating-date-time-values-manipulating-date-time-values)
+- [Current date-time moment](#functions-that-return-the-current-date-time-moment-current-date-time-moment)
+- [Delaying execution](#functions-for-delaying-execution-delaying-execution)
+- [Miscellaneous](#miscellaneous-functions-miscellaneous)
+
+<p id="functions-without-trailing-parentheses">Notice that the so-called _date-time_ formatting functions, like:</p>
```

Comment:
Instead of using **p** tags, it's preferable to use `<a name=""` tags instead to create named anchors. That way your paragraph content isn't tied up in HTML tags, which can be a nuisance for some searching and processing.

Reply:
All **p** tags changed to **a** tags. Yes, that's a big improvement. Wish I'd thought of that approach myself.